tests/compile-fail/function_calls/exported_symbol_bad_unwind3.rs

```diff
@@ -0,0 +1,16 @@
+#![feature(unwind_attributes)]
+#![feature(c_unwind)] // make sure it doesn't insert abort-on-unwind for the `#[unwind(allowed)]` function
```

Comment:
The feature flag should not affect what happens for `#[unwind(allowed)]` functions, should it?

Reply:
I think so. I'm just adding it here to make sure it doesn't.

Reply:
The comment is confusing though because it sounds like the feature flag would do that.
If the test still works without this feature flag, I think I'd prefer that.